messages/jp.json

```diff
@@ -144,7 +144,7 @@
 				"country": "国",
 				"phone": "電話番号",
 				"taxId": "税番号",
-				"taxIdPlaceholder": "例：PL123456789"
+				"taxIdPlaceholder": "例：T1234567890123"
```

Comment:
Considering as Japanese translation, I use the Japanese tax number format as an example.